addons/l10n_nz/data/account_tax_template_data.xml

```diff
@@ -280,24 +280,28 @@
             (0,0, {
                 'factor_percent': 100,
                 'repartition_type': 'base',
+                'plus_report_line_ids': [ref('tax_report_box11')],
             }),
 
             (0,0, {
                 'factor_percent': 100,
                 'repartition_type': 'tax',
                 'account_id': ref('nz_21330'),
+                'plus_report_line_ids': [ref('tax_report_box11')],
             }),
         ]"/>
         <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
             (0,0, {
                 'factor_percent': 100,
                 'repartition_type': 'base',
+                'minus_report_line_ids': [ref('tax_report_box11')],
```

Comment:
Ok for BOX 11, but remove the tags on the base line. This is a GST only tax, the base lines should be 0